bskyogcard/src/util/getStarterPackImageUri.ts

```diff
@@ -0,0 +1,13 @@
+import {AppBskyGraphDefs, AtUri} from '@atproto/api'
+
+const DEV = process.env.NODE_ENV === 'development'
+const HOST = DEV
+  ? `http://localhost:3000/start`
+  : `https://ogcard.cdn.bsky.app/start`
+
+export function getStarterPackImageUri(
+  record: AppBskyGraphDefs.StarterPackViewBasic,
+) {
+  const urip = new AtUri(record.uri)
+  return `${HOST}/${record.creator.did}/${urip.rkey}`
+}
```

Comment:
In local dev, this ensures we fetch SP images from the local server. In prod, it should hit the CDN. See NODE_ENV vars in package.json.